src/main/java/com/mycompany/SpringBootStarter.java

```diff
@@ -0,0 +1,15 @@
+package com.mycompany;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.servlet.ServletComponentScan;
+
+@SpringBootApplication
+@ServletComponentScan
```

Comment:
It's better to specify package for spring to scan (base package of your application). It will speed up app start.